pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -178,38 +178,48 @@ func NewCmdCreate(parentName string, f *clientcmd.Factory, out io.Writer) *cobra
 }
 
 const (
-	execLong = `Execute a command in a container`
-
-	execExample = `  # Get output from running 'date' in ruby-container from pod 'mypod'
-  %[1]s exec mypod -c ruby-container date
+	completionLong = `This command prints shell code which must be evaluated to provide interactive
+completion of %s commands.`
 
-  # Switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 'mypod' and sends stdout/stderr from 'bash' back to the client
-  %[1]s exec mypod -c ruby-container -i -t -- bash -il`
-
-	completionLong = `This command prints shell code which must be evaluation to provide interactive
-completion of openshift cli commands.`
-
-	completionExample = `  # Generate the openshift cli completion code for bash
-  source <(oc completion bash)
+	completionExample = `  # Generate the %s completion code for bash
+  %s completion bash > bash_completion.sh
+  source bash_completion.sh
```

Comment:
Ah, Xingxing Xia stated in the trello card for this PR [1] that "the behavior `source <(...)` does not work on Mac OS by design" [2]

[1] https://trello.com/c/r3EaiA9F/672-3-cli-oc-completion-command
[2] https://trello.com/c/r3EaiA9F/672-3-cli-oc-completion-command#comment-576ce0bafbadb62268db5234